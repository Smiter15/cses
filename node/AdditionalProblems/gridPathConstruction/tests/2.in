100
2 43 1 33 1 21
2 2 1 1 2 2
2 32 1 1 2 8
2 14 1 12 1 14
2 36 1 26 2 30
2 35 2 26 1 27
2 46 1 26 2 20
2 2 1 2 2 2
2 39 2 27 1 7
2 44 1 42 2 24
2 9 1 7 2 2
2 42 1 14 1 33
2 24 1 16 2 8
2 5 1 5 2 2
2 9 2 1 2 4
2 37 2 20 2 25
2 19 2 19 1 14
2 21 2 21 2 15
2 43 1 31 2 3
2 8 2 6 2 2
2 44 2 19 1 17
2 49 1 20 2 30
2 37 2 5 2 32
2 23 1 16 1 8
2 33 2 13 1 8
2 43 2 8 1 3
2 31 2 8 2 16
2 18 2 2 1 15
2 7 1 1 2 5
2 37 1 12 1 27
2 20 1 7 1 13
2 44 1 8 1 30
2 42 1 9 1 17
2 21 1 5 1 19
2 4 2 3 2 4
2 35 1 14 1 21
2 13 2 3 1 12
2 47 1 16 2 4
2 9 2 7 2 9
2 16 2 7 1 7
2 3 2 2 1 3
2 42 1 5 1 3
2 13 2 13 1 7
2 27 2 12 1 24
2 3 1 3 2 3
2 13 2 9 1 8
2 40 2 3 1 31
2 12 1 12 2 7
2 17 1 12 2 12
2 43 1 35 1 12
2 26 2 14 1 4
2 30 2 23 2 15
2 37 2 35 2 28
2 38 1 22 2 35
2 37 1 14 1 2
2 32 1 8 1 23
2 15 1 6 1 1
2 15 2 7 2 3
2 14 1 2 1 6
2 23 2 8 2 9
2 10 2 9 2 5
2 44 2 9 1 35
2 13 2 3 1 11
2 32 1 26 1 13
2 8 2 4 2 1
2 9 1 1 1 3
2 8 2 5 1 8
2 20 2 11 1 19
2 17 1 7 1 9
2 10 2 4 1 8
2 15 1 9 2 1
2 8 2 4 1 1
2 38 1 12 1 19
2 18 2 5 1 2
2 36 2 25 2 29
2 45 1 13 2 3
2 45 2 17 2 27
2 10 1 10 1 2
2 35 1 17 2 19
2 33 1 14 2 4
2 10 1 2 1 3
2 3 1 1 2 3
2 31 2 1 1 22
2 35 1 16 2 5
2 30 2 11 1 5
2 38 1 26 1 21
2 32 1 13 1 32
2 3 1 3 2 3
2 47 1 14 2 34
2 48 1 24 2 27
2 22 1 17 1 2
2 7 1 1 1 2
2 16 2 12 2 6
2 20 1 6 1 10
2 20 1 13 1 14
2 5 1 2 2 1
2 50 2 39 2 50
2 23 2 19 2 10
2 42 1 33 1 30
2 42 1 1 1 19
