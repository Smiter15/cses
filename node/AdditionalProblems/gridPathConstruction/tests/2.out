NO
NO
NO
NO
YES
LLLLLLLLLLLLLLLLLLLLLLLLLDRRRRRRRRRRRRRRRRRRRRRRRRRRURDRURRRRRRRDLLLLLL
NO
YES
RRRRRRRRRRRRRRRRRRRRDLLLLLLLLLLLLLLLLLLLLLULDLULDLULLLLLLLLLLLLLLLLLLLLDRRRRRRRRRRRRRRRRRRR
YES
LDR
YES
RRRRRRRRRRRRULLLLLLLLLLLLLDLULDLULDLULDLULDLULDLULDLULDLULDLULDLLLLLLLURRRRRR
YES
RRDLLLULDLULDLULDLULDLULDLULDLULDLULDLULLLLLLLLLLLLLLLLLLLLLLLLDRRRRRRRRRRRRRRRRRRRRRRR
NO
YES
LLLLLLLLLLLLLDRRRRRRRRRRRRRRURDRURDRURDRURDRURDRURDRURDRURDRURDRRRRRRRRRRULLLLLLLLL
YES
RRRRRRRRDLLLLLLLLLULDLULDLULDLULLLLLLLLDRRRRRRR
NO
YES
URDRURRRRRRDLLLLL
YES
LLLLLLLLLLLLLLLLLLLURRRRRRRRRRRRRRRRRRRRDRURDRURRRRRRRRRRRRRDLLLLLLLLLLLL
NO
NO
YES
RRRRRRRRRRRRDLLLLLLLLLLLLLULDLULDLULDLULDLULDLULDLULDLULDLULDLULDLULDLULDLULDLULLLDRR
NO
YES
RRRRRRRRRRRRRRRRRRRRRRRRRULLLLLLLLLLLLLLLLLLLLLLLLLLDLLLLLLLLLLLLLLLLLURRRRRRRRRRRRRRRR
YES
LLLLLLLLLLLLLLLLLLLDRRRRRRRRRRRRRRRRRRRRURDRURDRURDRURDRURRRRRRRRRRRRRRRRRRRRDLLLLLLLLLLLLLLLLLLL
YES
LLLLURRRRRDRURDRURDRURDRURDRURDRURDRURDRURDRURDRURDRURDRURDRURRRRRRDLLLLL
NO
NO
NO
NO
NO
YES
DRURDRURRRDLL
YES
LLLLLLLLLLLDRRRRRRRRRRRRURDRURDRURDRURDRURDRURDRURDRRRRRRRRRRRULLLLLLLLLL
NO
NO
NO
NO
YES
LLURRRD
YES
LLLLLLLLLLLLLDRRRRRRRRRRRRRRURDRURDRURDRRRRRRRRRRRRRRRULLLLLLLLLLLLLL
NO
YES
RRRRRRRRRRRRRRRRRRRRRRRRRRRRRRRDLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLULDLULDLULDLULDLULDLULLLLDRRR
NO
NO
NO
NO
YES
ULDLULDLULDLLLLLLLURRRRRR
YES
LLLLLLLLLLLURRRRRRRRRRRRDRURDRURDRURDRURDRURDRRRRULLL
YES
LLDRR
NO
YES
LLURRRDRURDRURDRURDRURDRURDRURDRURDRURDRURDRURDRURDRURDRURDRRRRRRRRRRULLLLLLLLL
NO
NO
YES
RRRRRRRRDLLLLLLLLLULDLULDLULDLULDLULDLULDLULDLULDLULDLULDLULDLLLLLLLLLLLLURRRRRRRRRRR
YES
RRRRRRRRRRRRULLLLLLLLLLLLLDLULDLULDLULDLULDLLLLURRR
NO
YES
RRULLLDLULDLULDLULLLLLLLLLLLLLLLLLLLLLLLLLLLLDRRRRRRRRRRRRRRRRRRRRRRRRRRR
NO
NO
YES
LLLLLLLDRRRRRRRRURDRURDRURDRURDRURDRURDRURDRRRRRRRRRRULLLLLLLLL
YES
RRRRRRRRRDLLLLLLLLLLULDLULDLU
NO
NO
YES
LLLLLLLURRRRRRRRRRRRRRRRRRRRRRDLLLLLLLLLLLLLL
NO
YES
LLLLLLLLURRRRRRRRRDRURDRURDRURDRURDRURDRURDRURDRURDRURDRURDRURDRURDRRRRRRRRRRULLLLLLLLL
YES
LLURRRDRURDRURDRURDRRRULL
YES
RRRRRRDLLLLLLLULDLULDLULDLULDLULDLULDLLLLLLLLLLLLLURRRRRRRRRRRR
YES
RRRRULLLLLDLULD
NO
NO
YES
LLLLLLLLLLURRRRRRRRRRRDRURDRURDRURDRRUL
NO
YES
LLLURRRRDRURDRRRULL
YES
RRRRRRDLLLLLLLULDLULDLULDLULD
NO
YES
LLLLLLLLLLLDRRRRRRRRRRRRURDRURDRURDRRRRRRRRRRRRRRRRRRRRULLLLLLLLLLLLLLLLLLL
NO
NO
YES
RRRRRRRRRRRRRRRRRRRRRRRRRRRRRRRRDLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLULDLULDLULDLULDLULLLDRR
NO
NO
YES
LLLLLLLLLLLLLLLLDRRRRRRRRRRRRRRRRRURRRRRRRRRRRRRRRRRDLLLLLLLLLLLLLLLL
YES
RRRRRRRRRRRRRRRRRRRDLLLLLLLLLLLLLLLLLLLLULDLULDLULDLULDLULLLLDRRR
YES
LDRRRRRRRRRULLLLLLL
YES
DRURD
NO
NO
YES
RRRRRRRRRRRRRRRRRRRULLLLLLLLLLLLLLLLLLLLDLULDLULDLLLLLURRRR
YES
RRRRRRRRRRRRDLLLLLLLLLLLLLULDLULDLLLLLLLLLLLLLLLLLLLLLURRRRRRRRRRRRRRRRRRRR
YES
LLLLLLLLLLLLDRRRRRRRRRRRRRURDRURDRURDRURDRURDRURDRURDRURDRURDRU
YES
LLDRR
YES
LLLLLLLLLLLLLDRRRRRRRRRRRRRRURDRURDRURDRURDRURDRURDRURDRURDRURDRURRRRRRRRRRRRRRDLLLLLLLLLLLLL
NO
YES
RRRRRDLLLLLLULDLULDLULDLULDLULDLULDLULDLLUR
YES
DRRRRRRULLLLL
NO
NO
YES
LLLLLLLLLLLLDRRRRRRRRRRRRRRRRRRRULLLLLL
NO
YES
LLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLURRRRRRRRRRRRRRRRRRRRRRRRRRRRRRRRRRRRRRRDRURDRURDRURDRURDRURD
YES
RRRRULLLLLDLULDLULDLULDLULLLLLLLLLLDRRRRRRRRR
YES
RRRRRRRRRDLLLLLLLLLLULDLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLURRRRRRRRRRRRRRRRRRRRRRRRRRRRR
NO
